pkg/account/component.go

```diff
@@ -304,3 +314,19 @@ func (c *component) MoveCredential(ctx context.Context, credentialID string, pre
 
 	return nil
 }
+
+func (c *component) GetConfiguration(ctx context.Context) (api.Configuration, error) {
+	var currentRealm = ctx.Value(cs.CtContextRealm).(string)
+
+	config, err := c.configDBModule.GetConfiguration(ctx, currentRealm)
+	if err != nil {
+		return api.Configuration{}, err
```

Comment:
Error is not yet "normalized"

Reply:
What do you mean with "normalized" ?

Reply:
Error codes we are using since recent _bsoniam_ changes